eng/pipelines/templates/steps/build-test.yml

```diff
@@ -83,11 +83,18 @@ steps:
           }
           Write-Host (Get-Command python).Source
 
+          if ($env:TESTMARKARGUMENT) {
```

Comment:
What is this needed for? I don't see it being used anywhere.

cc @JimSuplizio @alzimmermsft

Reply:
@weshaggard this is passed down through our new live platform matrix file - https://github.com/Azure/azure-sdk-for-python/pull/39390/files#diff-8913eb8c33079ee7e0e88901c8188996b2816dc5a369a86eb33874d75b26ee73
We wanted a different TestMarkArgument for each of the pipelines.

@scbedd can you please help answer this one in more detain in case I didn't explain it properly.

Reply:
Ok thanks for the context.